test/browser/spec/ol/render/webgl/shaderbuilder.test.js

```diff
@@ -3,18 +3,18 @@ import {
   colorToGlsl,
   numberToGlsl,
   stringToGlsl,
-} from '../../../../../src/ol/expr/gpu.js';
+} from '../../../../../../src/ol/expr/gpu.js';
 import {
   COMMON_HEADER,
   ShaderBuilder,
-} from '../../../../../src/ol/webgl/ShaderBuilder.js';
+} from '../../../../../../src/ol/render/webgl/ShaderBuilder.js';
 
 describe('ol.webgl.ShaderBuilder', () => {
   describe('getSymbolVertexShader', () => {
     it('generates a symbol vertex shader (with varying)', () => {
```

Comment:
With attributes now, no longer with varying.

Can you help me understand why this change in the tests was necessary?

Reply:
So the ShaderBuilder used to have two methods: `addAttribute` and `addVarying`. A prop value is transferred this way across vertex and fragment shaders:
```
      passed as attribute    passed as varying
             v                      v
propValue -------> vertex shader -------> fragment shader
```

Before this PR it was needed to call both `builder.addAttribute()` and `builder.addVarying()` to setup this pipeline, but now only `builder.addAttribute()` is required. This makes the API simpler and also simplifies a lot the library code downstream. I hope that cleared things up!

I'll update the test labels.

Reply:
Makes sense, thanks for the explanation.